spec/spec_helper.rb

```diff
@@ -1,6 +1,8 @@
 $LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
 require 'rspec'
 require 'authority'
+require 'pry-byebug'
+require 'pry'
```

Comment:
Nit: [`pry-byebug`](https://github.com/deivid-rodriguez/pry-byebug/blob/master/pry-byebug.gemspec#L25) should take care of installing `pry` for us